site/src/_layouts/default.hbs

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>d3fc {{#if title}}:{{/if}} {{title}}</title>
```

Comment:
Very pedantic, but if there's no title (index page) there will be some trailing whitespace. I think this ought to work:

```hbs
<title>d3fc{{#if title}} : {{title}}{{/if}}</title>
```

I also wonder if putting the title first, before _d3fc_, might be more useful?